lib_features.php

```diff
@@ -107,17 +107,15 @@ function send_support_mail($course, $text) {
 }
 
 function send_course_request_mail($recipient_username, $course, $request_id) {
-    global $USER, $CFG;
+    global $USER;
     $email = username_to_mail($recipient_username);
     $user = get_or_create_user($recipient_username, $email);
     $params = new stdClass();
     $params->a = $USER->firstname." ".$USER->lastname;
-    $params->b = $CFG->wwwroot.'/user/view.php?id='.$USER->id;
     $params->c = utf8_encode($course->titel);
-    $params->d = $CFG->wwwroot.'/local/lsf_unification/request.php?answer=12&requestid='.$request_id;
 
-    $data = array('userid' => $user->id, 'userfirstname' => $USER->firstname,
-        'userlastname' => $USER->lastname, 'params' => $params);
+    $data = array('userid' => $user->id, 'globaluserid' => $USER->id, 'userfirstname' => $USER->firstname,
```

Comment:
What's the difference between `userid` and `globaluserid`?

Reply:
user is the recipient and global user the sender, I will name them more precisely tomorrow.